### PR TITLE
some python3 fixes, one or two new functions

### PR DESCRIPTION
**Pull request by PeterMitrano**

### CODE REVIEW

**Comment by PeterMitrano** [comment]
need to rebase once #96 is merged

File: src/arc_utilities/color_mapping.py [hunk 1]
```diff
@@ -93,19 +93,23 @@ def rad_diff(x, y):
         return diff
 
 
-def adjust_hue((m_saturated, s_saturated, h_saturated), m_unsaturated):
```
Comment (LemonPi) [inline src/arc_utilities/color_mapping.py:96]: didn't know tuple parameter unpacking was a thing in python 2...

File: src/arc_utilities/extra_functions_to_be_put_in_the_right_place.py [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-from color_mapping import *
+import math
 
+from arc_utilities.color_mapping import interpolate_hot_to_cold
+from geometry_msgs.msg import Pose, Vector3, Point
+from std_msgs.msg import ColorRGBA
 
-def make_pose((px, py, pz), (rx, ry, rz, rw)):
+
+def make_pose(position, rotation):
```
Comment (LemonPi) [inline src/arc_utilities/extra_functions_to_be_put_in_the_right_place.py:8]: Why do we need these functions when we can use the constructor of the ROS messages?

**Review by LemonPi — APPROVED** [review]

File: src/arc_utilities/color_mapping.py (hunk 1 above)
Comment (PeterMitrano) [reply src/arc_utilities/color_mapping.py:96]: neither did I :upside_down_face:

File: src/arc_utilities/extra_functions_to_be_put_in_the_right_place.py (hunk 2 above)
Comment (PeterMitrano) [reply src/arc_utilities/extra_functions_to_be_put_in_the_right_place.py:8]: Not sure, and afaik they are not used in any of the repos in our org. I'll just delete them.

**Pull request merged**